src/backend/gl/src/info.rs

```diff
@@ -399,7 +399,7 @@ pub(crate) fn query_all(
         limits.max_viewports = get_usize(gl, glow::MAX_VIEWPORTS).unwrap_or(0);
     }
 
-    if info.is_supported(&[Core(4, 3), Ext("GL_ARB_compute_shader")]) {
+    if info.is_supported(&[Core(4, 3), Es(3, 2), Ext("GL_ARB_compute_shader")]) {
```

Comment:
Compute shaders have been available from `OpenGL ES 3.1`. If we require `OpenGL ES 3.0` context, should we add LegacyFeature for compute shaders?

Reply:
You are correct, compute is technically available on 3.1. However, the SSBOs require 3.2. Unless we can change the latter, I don't see a point in pretending we have compute shaders on 3.1. Am I missing anything?

Reply:
It would be great if we have a comment in code with an explanation of why we should have `OpenGL ES 3.2` for compute shaders 😄

Reply:
agreed, will do now